api/app/signals/apps/email_integrations/reporter/mail_actions.py

```diff
@@ -23,7 +28,8 @@
             'filters': {
                 'status__state__in': [GEMELD, ],
                 'reporter__email__isnull': False,
-                'reporter__email__gt': 0
+                'reporter__email__gt': 0,
+                'parent_id__exact': None,  # SIG-2931
```

Comment:
I would have doen `'parent_id__isnull': True,`, for all cases that need to check if they are a Signal with no children of a parent Signal.

Reply:
will do